SyntheticDataGeneration/Blender/GenerateLegoData.py

```diff
@@ -31,37 +31,103 @@
 IMAGE_Y_RESOLUTION = 64#0
 IMAGE_RENDER_MAX_CYCLES = 10#50
 TRAINING_BRICKS_FILE_NAMES = [
```

Comment:
How about we create an array with the set name ('London_21034') and add it to the array of training bricks?
Then we can add another array that would be our 'noise' of extra bricks and we keep the information of which part comes from which set.